src/Compilers/CSharp/Portable/CSharpResources.resx

```diff
@@ -682,7 +682,7 @@
     <value>'{0}': no suitable method found to override</value>
   </data>
   <data name="ERR_NamespaceUnexpected" xml:space="preserve">
-    <value>A namespace cannot directly contain members such as fields or methods</value>
+    <value>A namespace cannot directly contain members such as fields, methods or statements</value>
```

Comment:
This reads as statements being a kind of member

Reply:
what's a better term?  :)

Reply:
technically, this is how we represent it, GlobalStatementSyntax wraps a statement into a MemberDecl so we can have it in the compilation-unit/namespace-decl.